### DISCUSSION TITLE
Penpot 1.20.0 Docker Images

### DISCUSSION THREAD

**Discussion by georgkrause** [uxbox/uxbox#3904] [General]
Hello everyone, 

I noticed the Images for 1.20.0 are not available on Docker Hub. Did you changed the distribution channel or is there a problem with missing updates? 

Kind regards, Georg

**Comment by MDXDave** [comment]
There is no 1.20.0 release yet?

**Reply by georgkrause** [reply]
@MDXDave Stupid me, the [Changelog](https://github.com/penpot/penpot/blob/develop/CHANGES.md) confused me. Sorry for the noise!
